submissions/OwoeyeGbenga/landing_page/signup.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Glitz Stores</title>
+</head>
+<body>
+    <header class="w-full">
+        <div class="logo">
+            <img src="img/glitz.png" alt="Glitz" >
+        </div>
+    </header>
+    <form class="w-full">
+        <div class="container">
+          <h3>Sign Up</h3>
+          <hr>
+          <p>Please fill in this form to create an account.</p>
+          <label for="firstname">First Name:</label><br>
+          <input type="text" placeholder="First name" name="firstname" required><br/><br/>
+          <label for="lastname">Last Name:</label><br>
+          <input type="text" placeholder="Last name" name="lastname" required><br/><br/>
+          <label for="phonenumber">Phone Number:</label><br>
+          <input type="number" placeholder="Enter Phone" name="phonenumber"><br/><br/>
+          <label for="email">Email:</label><br>
+          <input type="text" placeholder="Enter Email" name="email" required><br/><br/>
+          <label for="psw">Password:</label><br>
+          <input type="password" placeholder="Enter Password" name="psw" required><br/><br/>
+          <label for="psw-repeat">Repeat Password:</label><br>
+          <input type="password" placeholder="Repeat Password" name="psw-repeat" required><br/><br/>   
+          <p>By creating an account you agree to our <a href="#">Terms & Privacy</a>.</p>
+          <button type="submit" class="registerBtn">Register</button>
+        </div>
+        <div class="containerSignin">
+          <p>Already have an account?<button type="submit" class="loginBtn">Log in</button></p>
+        </div>
+      </form>
```

Comment:
The form closing target doesn't align vertically with the opening one. Please fix.

Reply:
Done!!